topics/tas/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: tool-assisted-speedrun
+display_name: TAS
+short_description: Tool-assisted speedrunning is a type of video game speedrun, optimized with TAS software.
+topic: tas
+wikipedia_url: https://en.wikipedia.org/wiki/Tool-assisted_speedrun
+---
```

Comment:
thanks
b

Reply:
Please don't make unnecessary comments, they do nothing but ping me in GitHub notifications.